lua/telescope/builtin/__lsp.lua

```diff
@@ -452,7 +460,11 @@ local function get_workspace_symbols_requester(bufnr, opts)
       if client_res.error then
         vim.api.nvim_err_writeln("Error when executing workspace/symbol : " .. client_res.error.message)
       elseif client_res.result ~= nil then
-        vim.list_extend(locations, vim.lsp.util.symbols_to_items(client_res.result, bufnr))
+        if vim.fn.has "nvim-0.11" == 1 then
+          vim.list_extend(locations, vim.lsp.util.symbols_to_items(client_res.result, bufnr, "utf-8"))
+        else
+          vim.list_extend(locations, vim.lsp.util.symbols_to_items(client_res.result, bufnr))
+        end
```

Comment:
For the dynamic workspace symbol requester, the client id is the `_` variable in this for loop. So,
```lua
for client_id, client_res in pairs(results) do
	...
end
```